index.html

```diff
@@ -368,6 +368,25 @@ <h2>
                 <p>Return <var>p</var>.</p>
               </li>
             </ol>
+            <p>
+              The <a>user agent</a> MUST NOT capture content from the background of a
```

Comment:
(GitHub ate my draft comments...)
Can we clarify that background refers to what's behind the display surface, not the back part of said surface?

Reply:
I tried to retain the same language as much as possible to justify calling this a "move". But I'm happy to try to improve it. How about:
```suggestion
              The <a>user agent</a> MUST NOT capture content that's behind a
```

Reply:
> I tried to retain the same language as much as possible to justify calling this a "move". But I'm happy to try to improve it. How about:

IMHO that's clearer.